src/tests/suite/daos_rebuild.c

```diff
@@ -906,8 +907,8 @@ rebuild_multiple_tgts(void **state)
 	rebuild_io(arg, &oid, 1);
 
 	test_get_leader(arg, &leader);
-	daos_obj_layout_get(arg->coh, oid, &layout);
-
+	rc = daos_obj_layout_get(arg->coh, oid, &layout);
+	assert_int_equal(rc, 0);
```

Comment:
I realize this is a cherry-pick but it was missed.  This and others should be assert_rc_equal so we get a suitable error when the assertion fails.